projects/quasar/index.js

```diff
@@ -34,11 +31,36 @@ const lpStrategyContracts = {
     ],
 }
 
-
-async function tvl() {
+async function tvlOsmosis() {
     const api = new sdk.ChainApi({ chain: 'osmosis' })
 
     for (const poolID in lpStrategyContracts) {
+        let lpContracts = lpStrategyContracts[poolID];
+        for (const contractAddress of lpContracts) {
+            try {
+                let ica_balances = await queryContract({
+                    contract: contractAddress,
+                    chain: chain,
+                    data: { 'ica_balance': {} }
```

Comment:
I am trying to understand, what is this ica_balance? More important question, is there doublecounting between osmosis and quasar now?

Reply:
hmm, ica_balance is the balance of ica_address, which is the address of this contract in osmosis chain. I think it could be doublecounting,  so the real value lock is on osmosis pool so should we remove the quasar one?